cmd/main.go

```diff
@@ -103,7 +103,7 @@ func main() {
 				Action: func(c *cli.Context) error {
 					filter, err := qf.OpenReadOnlyFromPath(c.String("i"))
 					if err != nil {
-						return fmt.Errorf("can't read input file", err)
+						return fmt.Errorf("can't read input file: %w", err)
```

Comment:
nit: as we're changing the error messages we could also add some contextual information like this:

```
return fmt.Errorf("lookup: can't read input file: %w", err)
```